# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/08
# allow to use BioPython >= 1.83
diff -ru AlphaPulldown/alphafold/alphafold/data/mmcif_parsing.py AlphaPulldown-2.0.0b4_alphfold_SCOP.patch/alphafold/alphafold/data/mmcif_parsing.py
--- AlphaPulldown/alphafold/alphafold/data/mmcif_parsing.py	2024-08-17 15:34:23.000000000 +0200
+++ AlphaPulldown-2.0.0b4_alphfold_SCOP.patch/alphafold/alphafold/data/mmcif_parsing.py	2024-08-17 15:44:47.460653530 +0200
@@ -21,7 +21,7 @@
 
 from absl import logging
 from Bio import PDB
-from Bio.Data import SCOPData
+from Bio.Data import IUPACData as SCOPData
 
 # Type aliases:
 ChainId = str
fix import of protein_letters_3to1 from Biopython
author: Kenneth Hoste (HPC-UGent)
--- AlphaPulldown/alphapulldown/utils/remove_clashes_low_plddt.py.orig	2024-06-05 09:30:16.114746286 +0200
+++ AlphaPulldown/alphapulldown/utils/remove_clashes_low_plddt.py	2024-06-05 09:30:35.242665615 +0200
@@ -4,7 +4,7 @@
 from alphafold.data.mmcif_parsing import parse
 from alphafold.common.residue_constants import residue_atoms, atom_types
 from Bio.PDB import NeighborSearch, PDBIO, MMCIFIO
-from Bio.PDB.Polypeptide import protein_letters_3to1
+from Bio.Data.IUPACData import protein_letters_3to1
 from Bio import SeqIO
 from colabfold.batch import convert_pdb_to_mmcif
 from Bio.PDB import Structure, Model, Chain, Residue
